app/views/entities/_entity.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="mt-4" id="<%= dom_id entity %>">
+  <div class="d-flex justify-content-between entity_card p-4 m-4">
+    <p>
+      <strong>Category</strong>
+      <%= entity.name %>
+    </p>
+    <div>
+      <strong><%= entity.amount %></strong>
+    </div>
+  </div>
```

Comment:
### "Transactions page"

- [ ] For a given category, the list of transactions should be presented ordered by the most recent. kindly address this. 👇🏻 
![image](https://user-images.githubusercontent.com/79058364/234995321-7bf77dd7-7633-41d6-ba81-9c891e6db518.png)